src/sonic-config-engine/tests/sample_output/mvrf_interfaces

```diff
@@ -29,13 +29,10 @@ iface eth0 inet static
     up ip -4 route add default via 10.0.0.1 dev eth0 table 5000 metric 201
     up ip -4 route add 10.0.0.0/24 dev eth0 table 5000
     up ip -4 rule add from 10.0.0.100/32 table 5000
-    up cgcreate -g l3mdev:mgmt
-    up cgset -r l3mdev.master-device=mgmt mgmt
```

Comment:
Is it related to buster? If not, separate it into another PR?

Reply:
it is related.

Reply:
@lguohan Are these corrections because the 4.19 kernel has some issues? mgmt vrf must use cgexec command, which means mgmt vrf is not supported in 4.19 kernel?